docs/en/observability/logs-stream.asciidoc

```diff
@@ -133,20 +133,299 @@ image::images/stream-logs-example.png[example search query on the logs stream pa
+
+[source,console]
+----
+POST _ingest/pipeline/logs-example-default/_simulate
+{
+  "docs": [
+    {
+      "_source": {
+        "message": "2023-08-08T13:45:12.123Z WARN 192.168.1.101 Disk usage exceeds 90%."
+      }
+    }
+  ]
+}
+----
+
+The results should show the `@timestamp` field extracted from the `message` field:
+
+[source,console]
+----
+{
+  "docs": [
+    {
+      "doc": {
+        "_index": "_index",
+        "_id": "_id",
+        "_version": "-3",
+        "_source": {
+          "message": "WARN 192.168.1.101 Disk usage exceeds 90%.",
+          "@timestamp": "2023-08-08T13:45:12.123Z"
+        },
+        "_ingest": {
+          "timestamp": "2023-08-10T20:37:29.090624429Z"
+        }
+      }
+    }
+  ]
+}
+----
+
+NOTE: Create the index pipeline using the `PUT` command in the previous section before using the simulate pipeline API.
+
+[discrete]
+[[logs-stream-index-template]]
+==== Configure your data stream with an index template
+
+After creating your ingest pipeline, create an index template to point your log data to your pipeline using this command:
+
+[source,console]
+----
+PUT _index_template/logs-example-default-template
+{
+  "index_patterns": [ "logs-example-default" ],
+  "data_stream": { },
+  "priority": 500,
+  "template": {
+    "settings": {
+      "index.default_pipeline":"logs-example-default"
+    }
+  }
+}
+----
+
+Set the following values for the index template:
+
+- `index_patterns`– The index pattern needs to match your log data stream. Naming conventions for data streams are `<type>-<dataset>-<namespace>`. In this example, your logs data stream is named `logs-example-default`. Data that matches this pattern will go through your pipeline.
+- `data_stream` – Enables data streams.
+- `priority` – Index templates with higher priority take precedence over lower priority. If a data stream matches multiple index templates, the template with the higher priority is used. Built-in templates have a priority of `200`, so we recommend a priority higher than `200`.
+- `index.default_pipeline` – The name of your ingest pipeline. `logs-example-default` in this case.
+
+[discrete]
+[[logs-stream-create-data-stream]]
+==== Create your data stream
+
+Create your data stream using the {fleet-ref}/data-streams.html#data-streams-naming-scheme[data stream naming scheme]. The name needs to match the name of your pipeline. For this example, we'll name the data stream `logs-example-default` and use the example log:
+
+[source,console]
+----
+POST logs-example-default/_doc
+{
+  "message": "2023-08-08T13:45:12.123Z WARN 192.168.1.101 Disk usage exceeds 90%."
+}
+----
+
+Now look at the details using this command:
+
+[source,JSON]
+----
+GET /<index-name>/_doc/<document-id>
+----
+
+Under `_source`, you can see the `timestamp` field that was extracted by the dissect processor and the `@timestamp` field that was extracted by the date processor:
+
+[source,JSON]
+----
+{
+  "took": 0,
```

Comment:
For these kind of result documents, I suggest we filter out all the "non important" parts like `_shards` total hits etc. to make it focused on what matters in this case. Or do you think that is more confusing?

How can we indicate there is more content? For example using `...`

Reply:
++ we do this elsewhere in the docs too

Reply:
I was thinking about this as well. I'll see if we have any examples in our current docs to see if we have a consistent approach for this situation.

Reply:
I guess I was commenting on an outdated version of the PR.